### DISCUSSION TITLE
Keycloak Account theme - missing password.ftl theme in account v2,v3 themes

### DISCUSSION THREAD

**Discussion by lucas9122** [keycloak/keycloak#30915] [Archive]
There was a password.ftl template in keycloak account v1. Where can I find a similiar one in account.v2 or v3 themes ?

**Comment by jonkoops** [comment]
You can't, the new theme is a SPA so it only has `index.ftl` and the routing and pages are handled on the front-end.

**Comment by lucas9122** [comment]
@jonkoops 
ok, but when I extend v3, how can I override personal-info or navbar section. V2 has files responsible for it like AccountPage.js or PageNav.js, but default folder account v3 has only some minified files and I do not know what files should be override? or for instance I would like to remove a sidebar and navbar

**Reply by jonkoops** [reply]
In that case you'll have to create a custom React based theme, check out the [theme development guide](https://www.keycloak.org/docs/latest/server_development/#_themes) and [extending account console quickstart](https://github.com/keycloak/keycloak-quickstarts/tree/latest/extension/extend-account-console).

**Reply by lucas9122** [reply]
ok, but using "[extending account console quickstart](https://github.com/keycloak/keycloak-quickstarts/tree/latest/extension/extend-account-console)" - I can only add new components but can't override existing one, correct?

**Reply by jonkoops** [reply]
That is correct, we've been working on making it easier to create a custom themes, @edewit can probably tell you more about that effort.

**Reply by edewit** [reply]
Right you can use the account-ui as a dependency in your own project have a look at this [quickstart](https://github.com/keycloak/keycloak-quickstarts/tree/main/extension/extend-admin-console-node) that does just that

**Comment by lucas9122** [comment]
@edewit Thank you. Do you know how to easily. deploy my app to production ? my reace app creates two build files, main.css and main.js - which is included in index.ftl. I tried to run npm run build in dockerfile but it doesnt works. I read docs and I find out making jar file
